recipes/guiqwt/meta.yaml

```diff
@@ -0,0 +1,75 @@
+{% set name = "guiqwt" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fcad0c1745c2685d6dcc3367ba6c5fa5dbbcd5991bf005e787c36e8e49b520a2
+  fn: {{ name }}-{{ version }}.tar.gz
+
+  patches:
+    # Needed to avoid some random exceptions which can make the application
+    # crash
+    - curve.patch
+
+build:
+  number: 0
+  skip: true  # [win and vc!=14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+
+  host:
+    - setuptools
+    - pip
+    - python>=3
+    - qt=4.8.7
+    - pyqt=4.11.4
```

Comment:
According to the documentation it should work with ``pyqt 4 and 5``
```
PyQt4 4.x (x>=3 ; recommended x>=4) or PyQt5 5.x (x>=5)
```
So, please remove the pinning

Reply:
Will it create builds for both pyqt4 and 5 then?

Reply:
It doesn't matter, because ``conda`` will check just for ``pyqt``, this is a python package and as far I am concerned, it will not compile against ``pyqt``. So, that is ok to remove it

Reply:
Ooooh right right, thanks!